app/marketing/views.py

```diff
@@ -790,6 +790,24 @@ def leaderboard(request, key=''):
             for tech in techs:
                 technologies.add(tech)
 
+    import pandas
```

Comment:
Can you move this import to the top of the file?

Reply:
Actually, we don't need to use pandas for this - you could either take that list of country codes and set it up as a python file and just import that here. I think thats easiest.

Reply:
I've converted the csv into a python list and set the required flag variables inside the new file